deploy/helm-chart/templates/deployment-promscale.yaml

```diff
@@ -78,6 +78,7 @@ spec:
           configMap:
             name: {{ include "promscale.fullname" . }}
       serviceAccountName: {{ include "promscale.fullname" . }}
+      automountServiceAccountToken: false
```

Comment:
Could you also add this to `Service` object?

Reply:
Doesn't look like `automountServiceAccountToken` is a valid attribute to add to a `Service` object.